[2024-06-06T00:00:16.615+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-06T00:00:16.637+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestion_data.enviar_notificacion scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T00:00:16.650+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestion_data.enviar_notificacion scheduled__2024-06-05T00:00:00+00:00 [queued]>
[2024-06-06T00:00:16.651+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-06T00:00:16.669+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): enviar_notificacion> on 2024-06-05 00:00:00+00:00
[2024-06-06T00:00:16.676+0000] {standard_task_runner.py:63} INFO - Started process 2115 to run task
[2024-06-06T00:00:16.680+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingestion_data', 'enviar_notificacion', 'scheduled__2024-06-05T00:00:00+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/dag_entregable_fedebar.py', '--cfg-path', '/tmp/tmpc7yuz_bi']
[2024-06-06T00:00:16.682+0000] {standard_task_runner.py:91} INFO - Job 149: Subtask enviar_notificacion
[2024-06-06T00:00:16.741+0000] {task_command.py:426} INFO - Running <TaskInstance: ingestion_data.enviar_notificacion scheduled__2024-06-05T00:00:00+00:00 [running]> on host 167cb9f6b96a
[2024-06-06T00:00:16.846+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FedeBar' AIRFLOW_CTX_DAG_ID='ingestion_data' AIRFLOW_CTX_TASK_ID='enviar_notificacion' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-05T00:00:00+00:00'
[2024-06-06T00:00:16.848+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-06T00:00:16.859+0000] {base.py:84} INFO - Using connection ID 'mail_fede' for task execution.
[2024-06-06T00:00:20.542+0000] {weather_info_ETL.py:230} INFO - El email fue enviado correctamente.
[2024-06-06T00:00:20.543+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-06T00:00:20.543+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-06T00:00:20.552+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ingestion_data, task_id=enviar_notificacion, run_id=scheduled__2024-06-05T00:00:00+00:00, execution_date=20240605T000000, start_date=20240606T000016, end_date=20240606T000020
[2024-06-06T00:00:20.592+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-06T00:00:20.608+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-06T00:00:20.611+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
